scalding-db/src/main/scala/com/twitter/scalding/db/JdbcToHdfsCopier.scala

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2015 Twitter, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.twitter.scalding.db
+
+import com.twitter.scalding.JsonUtils
+
+import java.sql.{ DriverManager, ResultSet, ResultSetMetaData }
+import scala.util.{ Failure, Success, Try }
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{ FileSystem, FSDataOutputStream, Path }
+import org.slf4j.LoggerFactory
+
+object JdbcToHdfsCopier {
+
+  protected val log = LoggerFactory.getLogger(this.getClass)
+
+  def apply[T <: AnyRef: Manifest](connectionConfig: ConnectionConfig,
```

Comment:
can you add a comment why we need the Manifest? It looks like we are converting to JSON. Why is that? Why not take an Injection in writeToHdfs and then set that injection up here or let the user supply?